travel-event-host-app/src/lib/validators/rules/password-complexity-rule.ts

```diff
@@ -3,15 +3,12 @@ import { ValidationError } from '../validation-error/validation-error';
 
 export class PasswordComplexityRule extends BaseRule {
   constructor(fieldName: string) {
-    super(
-      'Password should be at least 8 characters and contain at least one number, one uppercase letter, one lowercase letter and a special character.',
-      fieldName,
-    );
+    super('Password should be at least 8 characters.', fieldName);
   }
 
   validate(value: string): ValidationError | undefined {
     const passwordRegex = /((?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%]).{8,30})/;
-    if (!passwordRegex.test(value)) {
+    if (value[0] === undefined || value[0].trim().length < 8) {
```

Comment:
Maybe you can ask the team what they think about the password complexity rules.

Reply:
yes, btw it was working with errors, but changing the complexity is my bad